src/System.Devices.Gpio/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -98,21 +97,33 @@ private static void Main(string[] args)
                     case 20:
                         Unix_ButtonWait();
                         break;
-
                     case 21:
                         RaspberryPi_ButtonWait();
                         break;
 
                     case 22:
-                        Unix_LiquidCrystal();
+                        Unix_LCD();
```

Comment:
Can you make this "Lcd" to be consistent with how you've cases Spi and Bme? 

https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx

Reply:
Yes, sorry I didn't realized about this.